eclipse/plugin/META-INF/MANIFEST.MF

```diff
@@ -34,6 +34,7 @@ Import-Package: com.puppycrawl.tools.checkstyle;resolution:=optional,
  org.eclipse.jface.wizard,
  org.eclipse.swt,
  org.eclipse.swt.awt,
+ org.eclipse.swt.browser,

```

Comment:
Note for myself: Take a look. Was this really necessary?

Reply:
Answer to myself: no it was not :joy:

Reply:
probably the link to open the browser ...